com.unity.render-pipelines.core/Runtime/Common/CoreAttributes.cs

```diff
@@ -13,4 +13,12 @@ public class DisplayInfoAttribute : Attribute
         /// <summary>Display order used in UI.</summary>
         public int order;
     }
+
+    /// <summary>
+    /// Attribute used to customize UI display to allow properties only be visible when "Show Additional Properties" is selected
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Field)]
```

Comment:
I'm wondering if we should call this attribute "AdditionalProperty". "Additional" alone seems to vague.

Reply:
Perfect, I will rename it to `AdditionalProperty`. Thanks